src/background.ts

```diff
@@ -2,7 +2,7 @@ import { MyStorage, SearchEngine, CurrentState, ENGINES, isUrlSupported, TypedMs
 
 
 const STORAGE: MyStorage = {
-    enabledEngines: [ "duckduckgo", "startpage", "bing", "google" ]
+    enabledEngines: [ "duckduckgo", "startpage", "bing", "google", "enwiki" ]
```

Comment:
I think this should not be added into default setting, because:
1. it's not a general search engine.
2. this is for English only.

Reply:
I removed it from default.

As mentioned I also would implement an option to use other languages. Adding a configuration interface that allows to add arbitrary pages would be nice but this would be a bigger project taking much longer. For now it would therefore be nice to add these options.